src/momento/internal/aio/_scs_data_client.py

```diff
@@ -772,6 +793,160 @@ async def set_remove_elements(
             self._log_request_error("set_remove_elements", e)
             return CacheSetRemoveElements.Error(convert_error(e))
 
+    async def sorted_set_put_elements(
+        self,
+        cache_name: TCacheName,
+        sorted_set_name: TSetName,
+        elements: TSortedSetElements,
+        ttl: CollectionTtl = CollectionTtl.from_cache_ttl(),
+    ) -> CacheSortedSetPutElementsResponse:
+        try:
+            self._log_issuing_request("SortedSetPutElements", {})
+            _validate_cache_name(cache_name)
+            _validate_sorted_set_name(sorted_set_name)
+
+            request = _SortedSetPutRequest()
+            request.set_name = _as_bytes(sorted_set_name, self.__UNSUPPORTED_SORTED_SET_NAME_TYPE_MSG)
+            for value, score in _gen_sorted_set_elements_as_bytes(
+                elements, self.__UNSUPPORTED_SORTED_SET_ELEMENTS_TYPE_MSG
+            ):
+                _validate_sorted_set_score(score)
+                element = _SortedSetElement()
+                element.value = value
+                element.score = score
+                request.elements.append(element)
+            request.ttl_milliseconds = self._collection_ttl_or_default_milliseconds(ttl)
+            request.refresh_ttl = ttl.refresh_ttl
+
+            await self._build_stub().SortedSetPut(
+                request,
+                metadata=make_metadata(cache_name),
+                timeout=self._default_deadline_seconds,
+            )
+            self._log_received_response("SortedSetPutElements", {"sorted_set_name": str(request.set_name)})
+            return CacheSortedSetPutElements.Success()
+        except Exception as e:
+            self._log_request_error("sorted_set_put_elements", e)
+            return CacheSortedSetPutElements.Error(convert_error(e))
+
+    async def sorted_set_fetch_by_score(
+        self,
+        cache_name: TCacheName,
+        sorted_set_name: TSortedSetName,
+        min_score: float | None,
+        max_score: float | None,
+        sort_order: SortOrder,
+        offset: int | None,
+        count: int | None,
+    ) -> CacheSortedSetFetchResponse:
+        try:
+            self._log_issuing_request("SortedSetFetch", {"sorted_set_name": str(sorted_set_name)})
+            _validate_cache_name(cache_name)
+            _validate_sorted_set_name(sorted_set_name)
+
+            request = _SortedSetFetchRequest()
+            request.set_name = _as_bytes(sorted_set_name, "Unsupported type for set_name: ")
+            request.with_scores = True
+
+            if min_score is not None:
+                request.by_score.min_score = min_score
+            else:
+                request.by_score.unbounded_min.CopyFrom(_Unbounded())
```

Comment:
Is `CopyFrom` necessary or can we just directly assign `_Unbounded()` to `unbounded_min`